cli/src/main.rs

```diff
@@ -203,15 +203,26 @@ fn find_configuration(args: &Args) -> Result<PathBuf> {
     Ok(config_path)
 }
 
+pub fn print_error_as_warning(error: &anyhow::Error) {
+    warn!("An error occurred. Resuming...");
+    for cause in error.chain() {
+        warn!(" |- {}", cause);
+    }
+}
+
+pub fn print_error(error: &anyhow::Error) {
+    error!("An error occurred:");
+    for cause in error.chain() {
+        error!(" |- {}", cause);
+    }
+}
```

Comment:
We shouldn't need two functions. How about just
```rust
pub fn print_error(error: &anyhow::Error) {
    for cause in error.chain() {
        error!(" |- {}", cause);
    }
}
```
and leave the call to `error!()`before it in `main()` below. You can add the call to `warn!()` in `upload_batch_of_annotations()`, but I'm not sure logging in that case is necessary.